bears/general/LineCountBear.py

```diff
@@ -18,24 +19,58 @@ def _get_blank_line_count(self, file):
         return num_blank_lines
 
     def run(self, filename, file, max_lines_per_file: int,
-            exclude_blank_lines: bool=False):
+            ignore_files_regex: typed_list(str),
```

Comment:
there's `ignore_files` defined by coala's settings system. You shouldn't implement such settings for bears.

Reply:
@Makman2 Hm, so I shouldn't consider exceptions for required limit? :) I will need ignore_files for check exceptions later :)